src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
 public class WorkWithFile {
+    private static final String SPLIT_STRING = ",";
+    private static final String STRING_BUY = "buy";
+    private static final String STRING_SUPPLY = "supply";
+    private static final String STRING_RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        ArrayList<String> data = readFomCsv(fromFileName);
```

Comment:
```suggestion
        List<String> data = readFomCsv(fromFileName);
```
We always use an interface when declaring variable.